src/main/scala/cromwell/engine/backend/jes/JesAttributes.scala

```diff
@@ -23,6 +23,7 @@ object JesAttributes {
     "baseExecutionBucket",
     "endpointUrl",
     "authenticationMode",
+    "maximumPollingInterval",
```

Comment:
The option was added in a previous commit. Somehow it wasn't included in the "expected options" list.